packages/puppeteer-core/src/api/Page.ts

```diff
@@ -1679,21 +1685,16 @@ export class Page extends EventEmitter {
    *   await browser.close();
    * })();
    * ```
-   *
-   * @remarks List of all available devices is available in the source code:
```

Comment:
let's still link to the full list of devices

Reply:
ah is the link in the beginning now?

Reply:
Yes, the link is in the beginning now :)